tests/TestWorker.py

```diff
@@ -249,10 +249,9 @@ def test_InvalidEncoding(self):
         """Test Worker using Action.InvalidEncodingAction"""
         self.action_test_helper('invalid.encoding', 0, 0, 4096, 2, 3, 2)
 
+    @unittest.skipUnless('nt' == os.name, 'skipping on non-Windows')
```

Comment:
This doesn't look like a Python 3 thing, but the decorator is more elegant. I hadn't noticed that. Thanks.